src/buildConfig.js

```diff
@@ -119,7 +123,7 @@ function addIgnores(config, configDir, ignorePath) {
   // Absoluteize config.ignoreFiles
   if (config.ignoreFiles) {
     config.ignoreFiles = [].concat(config.ignoreFiles).map(glob => {
-      if (path.isAbsolute(glob)) return glob
+      if (path.isAbsolute(glob.replace(/^\!/, ""))) return glob
```

Comment:
`path.isAbsolute` will return `false`, when `glob === "!E:\myproj\**\*.css"`

In some case of `/stylelint/src/__tests__/ignore-test.js`, one configuration object will use for two files, so this object will pass to `buildConfig` twice.
In first time `config.ignoreFiles` will convert to absolute paths.
In second time `config.ignoreFiles` maybe have "!E:\myproj\**\*.css"`.